README.md

```diff
@@ -12,25 +12,44 @@ $ npm start
 Start Firefox in remote debugging mode and go to a tab you want to debug.
 
 ```
-$ /Applications/FirefoxNightly.app/Contents/MacOS/firefox-bin --start-debugger-server 6080
+npm run start-firefox
 ```
 
+NOTE: you might need to enable two settings in your browsers settings. Go to `about:config` and set `devtools.debugger.remote-enabled`, `devtools.chrome.enabled` to true.
+
 Go to the Debugger!
 
 ```
-$ open index.html
+$ localhost:8000
 ```
 
 
 #### Advanced
 
-##### User Configuration
+##### User Settings
 
-You can add an `environment.json` to set user environmental variables, like the firefox source path. Start by copying the environment.sample.
+You can add an `environment.json` to customize your local environment. Start by copying `environment.json.sample`.
 
 ```json
-{ "firefoxSrcPath" : "/Users/jlaster/src/mozilla/gecko-dev"}
+{
+  "firefoxBinDir": "/Applications/FirefoxDeveloperEdition.app/Contents/MacOS/firefox-bin"  
+}
+```
+
+#### Quickly Running Firefox
+
+If you have [Firefox DeveloperEdition](https://www.mozilla.org/en-US/firefox/developer/) installed, you can easily start firefox with this command `npm run start-firefox`.
+
+You can also run start firefox nightly, by changing the `firefoxBinDir` field in environment.json.
+
+NOTE: for some reason `npm run start-firefox` fails when trying to start Firefox.app.
```

Comment:
That's a very weird edge case. Is it worth having `start-firefox` if it only works some of the time?